sdk/typescript/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@lfdecentralizedtrust-labs/paladin-sdk",
-  "version": "0.0.4-alpha.1",
+  "version": "0.0.4-alpha.2",
```

The current released version is `0.0.5`, since this is not yet released as `0.0.6` it should be. the "alpha" release for `0.0.6`
```suggestion
  "version": "0.0.6-alpha.1",
```